app/models/ApplicationRequest.scala

```diff
@@ -56,67 +67,137 @@ case class EORIDetails(
+    extends AnyVal
+    with IdenticalGoodsExplaination
+object OtherUsersIdenticalGoods {
+  implicit val format = Json.valueFormat[OtherUsersIdenticalGoods]
+}
+
+object IdenticalGoodsExplaination {
+  import ApplicationRequest.jsonConfig
+  implicit val format: Format[IdenticalGoodsExplaination] =
+    Json.configured(jsonConfig).format[IdenticalGoodsExplaination]
+}
 
 case class MethodTwo(
   whyNotOtherMethods: String,
   detailedDescription: IdenticalGoodsExplaination
 ) extends RequestedMethod
+object MethodTwo {
+  implicit val format: OFormat[MethodTwo] = Json.format[MethodTwo]
+}
 
-sealed trait SimilarGoodsExplaination
-// again, "_type": "PreviouslyImportedGoods", etc
-case class PreviousSimilarGoods(value: String) extends SimilarGoodsExplaination
-case class OtherUsersSimilarGoods(value: String) extends SimilarGoodsExplaination
+sealed trait SimilarGoodsExplaination extends Any
+case class PreviousSimilarGoods(val value: String) extends AnyVal with SimilarGoodsExplaination
+case class OtherUsersSimilarGoods(val value: String) extends AnyVal with SimilarGoodsExplaination
+
+object PreviousSimilarGoods {
+  implicit val format: Format[PreviousSimilarGoods] = Json.valueFormat[PreviousSimilarGoods]
+}
+object OtherUsersSimilarGoods {
+  implicit val format: Format[OtherUsersSimilarGoods] = Json.valueFormat[OtherUsersSimilarGoods]
+}
+
+object SimilarGoodsExplaination {
+  import ApplicationRequest.jsonConfig
+  implicit val format: OFormat[SimilarGoodsExplaination] =
+    Json.configured(jsonConfig).format[SimilarGoodsExplaination]
+}
 
 case class MethodThree(
   whyNotOtherMethods: String,
   detailedDescription: SimilarGoodsExplaination
 ) extends RequestedMethod
+object MethodThree {
+  implicit val format: Format[MethodThree] =
+    Json.format[MethodThree]
+}
 
 case class MethodFour(
   whyNotOtherMethods: String,
   deductiveMethod: String
 ) extends RequestedMethod
+object MethodFour {
+  implicit val format: OFormat[MethodFour] = Json.format[MethodFour]
+}
 
 case class MethodFive(
-  whyNotOtherMethods: Boolean,
+  whyNotOtherMethods: String,
   computedValue: String
 ) extends RequestedMethod
+object MethodFive {
+  implicit val format: OFormat[MethodFive] = Json.format[MethodFive]
+}
+
+sealed abstract class AdaptedMethod(override val entryName: String) extends EnumEntry
+
+object AdaptedMethod extends Enum[AdaptedMethod] with PlayJsonEnum[AdaptedMethod] {
+  val values: IndexedSeq[AdaptedMethod] = findValues
+
+  case object MethodOne extends AdaptedMethod("MethodOne")
+  case object MethodTwo extends AdaptedMethod("MethodTwo")
+  case object MethodThree extends AdaptedMethod("MethodThree")
+  case object MethodFour extends AdaptedMethod("MethodFour")
+  case object MethodFive extends AdaptedMethod("MethodFive")
+  case object Unable extends AdaptedMethod("Unable")
+}
 
 case class MethodSix(
   whyNotOtherMethods: String,
-  adoptMethod: AdoptMethod,
+  adoptMethod: AdaptedMethod,
   valuationDescription: String
```

Comment:
Field `valuationDescription` is a String but MethodTwo and MethodThree use case classes for similar fields. Is that intentional?

Reply:
Yep, MethodSix just is always the same three questions so it can just be a string here

Methods Two/Three require the classes as the questions branch and need a way to determine the route used